zebra/zebra_evpn_mh.c

```diff
@@ -3369,7 +3370,7 @@ DEFPY(zebra_evpn_es_sys_mac,
 /* CLI for setting up local-ID part of ESI on an access port */
 DEFPY(zebra_evpn_es_id,
       zebra_evpn_es_id_cmd,
-      "[no$no] evpn mh es-id [(1-16777215)$es_lid | NAME$esi_str]",
+      "[no$no] evpn mh es-id <(1-16777215)$es_lid | NAME$esi_str>",
```

Comment:
With this change, no form won't work without argument as before.

Reply:
Done. Sorry for it.